packages/editor/src/lib/editor/Editor.ts

```diff
@@ -3885,6 +3895,24 @@ export class Editor extends EventEmitter<TLEventMap> {
 		return this.store.get(typeof asset === 'string' ? asset : asset.id) as TLAsset | undefined
 	}
 
+	async resolveAssetUrl(
+		assetId: TLAssetId | null,
+		context: { screenScale: number; steppedScreenScale: number }
```

Comment:
could we make the signature here just take `screenScale` and derive `steppedScreenScale` here instead?